services/GoogleAdSense.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Google AdSense",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://www.google.com/adsense/tc/2018/UnitedKingdom.html",
+      "select": "body"
+    },
+    "Program Policies": {
+      "fetch": "https://support.google.com/adsense/answer/48182",
+      "select": ".article-container",
+      "remove": ".action-button, .zippy"
+    },
+    "Parent Organization Privacy Policy": {
+      "fetch": "https://policies.google.com/privacy?hl=en-US",
```

Comment:
There still are noises on this document, at least SVG images and Table of content.

Reply:
I copy-pasted the Google document and consequently didn't check it. Don't we want to keep images ? 
Will take care of it in a separate PR to clean Google if you don't mind :)

Reply:
We only want to keep images that have a meaning in the terms, not illustrations.

Reply:
Finally solved, I didn't open another PR.